src/Symfony/Component/WebLink/Link.php

```diff
@@ -0,0 +1,153 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\WebLink;
+
+use Psr\Link\EvolvableLinkInterface;
+
+class Link implements EvolvableLinkInterface
+{
+    // Relations defined in https://www.w3.org/TR/html5/links.html#links and applicable on link elements
+    const REL_ALTERNATE = 'alternate';
+    const REL_AUTHOR = 'author';
+    const REL_HELP = 'help';
+    const REL_ICON = 'icon';
+    const REL_LICENSE = 'license';
+    const REL_SEARCH = 'search';
+    const REL_STYLESHEET = 'stylesheet';
+    const REL_NEXT = 'next';
+    const REL_PREV = 'prev';
+
+    // Relation defined in https://www.w3.org/TR/preload/
+    const REL_PRELOAD = 'preload';
+
+    // Relations defined in https://www.w3.org/TR/resource-hints/
+    const REL_DNS_PREFETCH = 'dns-prefetch';
+    const REL_PRECONNECT = 'preconnect';
+    const REL_PREFETCH = 'prefetch';
+    const REL_PRERENDER = 'prerender';
+
+    // Extra relations
+    const REL_MERCURE = 'mercure';
```

Comment:
We should define the constants as `public`.

Reply:
they're already public :)

Reply:
I know, that's purely CS. I didn't know if we have a policy on defining constant visibility yet.